ledger/ledger-api-bench-tool/src/main/scala/com/daml/ledger/api/benchtool/metrics/MetricsManager.scala

```diff
@@ -13,9 +13,14 @@ object MetricsManager {
 
   sealed trait Message[T]
```

Comment:
I need the type parameter to be able to abstract over stream element type in metrics. Metrics expose `def onNext(value: T)` method which is in turn used by the `MetricsManager[T]`.

Reply:
But you don't use the type downstream, it's unchecked

Reply:
Do you propose instead of having `Message.NewValue[T](value: T)` to have `Message.NewValue(value: Any)`?

Reply:
Example https://github.com/digital-asset/daml/commit/56f006043abced201c3685e7d1c987ee5eac183a

Reply:
>Do you propose instead of having Message.NewValue[T](value: T) to have Message.NewValue(value: Any)?

No, you can have the type parameter where it is needed: only for `NewValue[T]`. The other case classes don't need it

Reply:
You're right! Thank you :)